.github/workflows/release.yaml

```diff
@@ -0,0 +1,47 @@
+# Builds and pushes a docker image on a new tag.
+#
+# Uses the action:
+# https://github.com/marketplace/actions/build-and-push-docker-images
+name: ci
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
You don't need to set up qemu unless you're planning on building multi-architecture images.

Reply:
I decided on leaving this here initially, and seeing how much time it adds. Depending on what we do with the Power9s, we may or may not want multiarch builds yet.